### DISCUSSION TITLE
Set-PnpTraceLog - Console logging not supported

### DISCUSSION THREAD

**Discussion by pmatthews05** [pnp/powershell#258] [Q&A]
Slowly updating my PNPs script to use this Core version of PNP.

When I set my command
```ps
Set-PnPTraceLog -On -Level:Debug
```
I get the error message "WARNING: Console logging not supported"

Even trying the command
```ps
Set-PnPTraceLog -On -Level:Debug -WriteToConsole
```
I still get the error message.

Has this functionality been removed in the new version of PNP?

**Comment by erwinvanhunen** [comment]
No, it hasn't been removed, but I now notice it too. It seems to be a limitation (at first sight) of the .NET Core framework we switched to, but I will have to investigate that more. File logging still works as an alternative (-LogFile parameter)

**Comment by erwinvanhunen** [comment]
Move this to an issue: https://github.com/pnp/powershell/issues/267
